townhall/myapi/dao.py

```diff
@@ -251,3 +253,44 @@ def filtered_organization(
             filters["location__icontains"] = filtered_organization_data.location
 
         return Organization.objects.filter(**filters)
+
+class TaskDao:
+
+    @staticmethod
+    def get_all_tasks():
```

Comment:
please add type hinting to all these methods

Reply:
Done